modules/programs/exa.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.exa;
+
+  aliases = {
+    ls = "${pkgs.exa}/bin/exa";
+    ll = "${pkgs.exa}/bin/exa -l";
+    la = "${pkgs.exa}/bin/exa -a";
+    lt = "${pkgs.exa}/bin/exa --tree";
+    lla = "${pkgs.exa}/bin/exa -la";
+  };
+
+in {
+  meta.maintainers = [ maintainers.kalhauge ];
+
+  # Exa is modern replacement for 'ls'
+  # https://github.com/ogham/exa
+  options.programs.exa = {
+    enable = mkEnableOption "exa";
+
+    enableAliases = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Whether to enable recommended exa aliases.
+      '';
+    };
```

Could use `mkEnableOption`:
```suggestion
    enableAliases = mkEnableOption "recommended exa aliases";
```

---

Great point! Maybe this should propagate to https://github.com/nix-community/home-manager/blob/master/modules/programs/lsd.nix#L25 where most of this code is from?